common/lib/xmodule/xmodule/tests/test_license.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```

Comment:
TODO: This test file needs MAJOR refactoring. There's a lot of repeated code. Should use DDT to drive testing of repeatable elements.

I've done a minor refactoring already to make this more Pythonic.